### DISCUSSION TITLE
Can I use it for web?

### DISCUSSION THREAD

**Discussion by saiftheboss7** [TelemetryDeck/Viewer#80] [Q&A] [answered]
Seems pretty cool. Can I use it on a web app instead of iOS/Android apps? Like I want to send events from my PHP based website.

**Comment by winsmith** [comment, marked as answer]
Yes you can! ☺️ Here is a [Tutorial on using AppTelemetry via JavaScript](https://apptelemetry.io/pages/website-apptelemetry.html)
